.changeset/clean-moose-cover.md

```diff
@@ -0,0 +1,6 @@
+---
+"@rocket.chat/meteor": minor
+"@rocket.chat/i18n": minor
+---
+
+This feature introduces the use of the API_User_Limit whenever we set a channel as to be autojoined in a team, so it follows a config value insted of a hard-coded value of 50 (which could look like a bug)
```

Comment:
```suggestion
Introduced the use of the `API_User_Limit`setting to limit amount of members to simultaneoulsy auto-join a room in a team
```
Sorry for nitpicking, but we usually write changesets in the past tense and keep them small (just so that we have a pattern for the release notes). We have a guide [here](https://developer.rocket.chat/contribute-to-rocket.chat/modes-of-contribution/participate-in-rocket.chat-development/development-workflow#4.-adding-changeset-to-your-pull-request)